sdk/servicebus/service-bus/src/serviceBusAtomManagementClient.ts

```diff
@@ -1694,7 +1694,7 @@ export class ServiceBusAdministrationClient extends ServiceClient {
     subscriptionName: string,
     operationOptions?: OperationOptions
   ): Promise<boolean> {
-    const { span, updatedOperationOptions } = createSpan(
+    const { span, updatedOptions } = createSpan(
```

Comment:
@HarshaNalluru  - basically all of our changes were just renames because the field name is different, but the contents and everything else will work the same.